### ISSUE TITLE
Installation Instructions for Running the Tutorial Are Out of Date

### ISSUE THREAD

**Issue by CraigMiloRogers** [usc-isi-i2/kgtk#674]
In the following link:
```
https://kgtk.readthedocs.io/en/latest/install/
```

in the following section:
```
In your kgtk-notebooks folder, start the first KGTK Tutorial notebook using the following command:
```

The following command fails:
```
jupyter lab part1-kgtk-intro.ipynb
```

**Comment by CraigMiloRogers** [comment]
Perhaps the following is correct?
```
jupyter lab tutorial/01-kgtk-introduction.ipynb
```

**Comment by saggu** [comment]
@CraigMiloRogers I think you are right, we changes the directory structure in kgtk-notebooks repo. Instructions are outdated.

**Comment by saggu** [comment]
I'll make the change

**Comment by saggu** [comment]
Fixed in commit 9d5798f324c8db7eab3a0bfe3ce2dc941c94ece8
